### ISSUE TITLE
How can the typeof `nace_classification$code`  be a doble?

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/r2dii.data#185]
``` r
library(dplyr, warn.conflicts = FALSE)
library(r2dii.data)


data_dictionary %>% 
  filter(column == "code", dataset == "nace_classification") %>% 
  select(typeof)
#> # A tibble: 1 x 1
#>   typeof
#>   <chr> 
#> 1 double

# This can't be a double
nace_classification %>% 
  filter(code == "A")
#> # A tibble: 1 x 5
#>   original_code code  code_level sector       borderline
#>   <chr>         <chr>      <dbl> <chr>        <lgl>     
#> 1 A             A              1 not in scope FALSE

# So it isn't a double -- it is a character
typeof(nace_classification$code)
#> [1] "character"
```

<sup>Created on 2020-11-30 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by maurolepore** [comment]
@jdhoffa, should we remove the row where `code == "A"`? (And any other that can't be coerced to a double?)

**Comment by maurolepore** [comment]
This might not be an issue with 0.1.4 but with the version in development.

**Comment by maurolepore** [comment]
``` r
library(dplyr, warn.conflicts = FALSE)
library(remotes)
library(withr)
library(fs)

temp_lib <- tempdir()
fs::dir_create(temp_lib)

withr::with_libpaths(temp_lib)
#> Error in force(code): argument "code" is missing, with no default

install_version("r2dii.data", version = "0.1.4")
#> Downloading package from url: https://packagemanager.rstudio.com/all/__linux__/bionic/latest/src/contrib/r2dii.data_0.1.4.tar.gz
#> Installing package into '/home/mauro/R/x86_64-pc-linux-gnu-library/4.0'
#> (as 'lib' is unspecified)
#> Adding 'r2dii.data_0.1.4_R_x86_64-pc-linux-gnu.tar.gz' to the cache
packageVersion("r2dii.data")
#> [1] '0.1.4'

release <- r2dii.data::nace_classification %>%
  filter(code == "A")

typeof(release$code)
#> [1] "double"
```

<sup>Created on 2020-11-30 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

--

``` r

library(dplyr, warn.conflicts = FALSE)
library(remotes)
library(withr)
library(fs)

temp_lib <- tempdir()
fs::dir_create(temp_lib)

withr::local_libpaths(temp_lib)
#> Setting deferred event(s) on global environment.
#>   * Execute (and clear) with `withr::deferred_run()`.
#>   * Clear (without executing) with `withr::deferred_clear()`.

install_github("2DegreesInvesting/r2dii.data")
#> Using github PAT from envvar GITHUB_PAT
#> Downloading GitHub repo 2DegreesInvesting/r2dii.data@HEAD
#>      checking for file ‘/tmp/RtmpcFGoC0/remotes755e685b7803/2DegreesInvesting-r2dii.data-be2f0e6/DESCRIPTION’ ...  ✓  checking for file ‘/tmp/RtmpcFGoC0/remotes755e685b7803/2DegreesInvesting-r2dii.data-be2f0e6/DESCRIPTION’
#>   ─  preparing ‘r2dii.data’:
#>      checking DESCRIPTION meta-information ...  ✓  checking DESCRIPTION meta-information
#>   ─  checking for LF line-endings in source and make files and shell scripts
#>   ─  checking for empty or unneeded directories
#>    Removed empty directory ‘r2dii.data/vignettes’
#>   ─  building ‘r2dii.data_0.1.4.9000.tar.gz’
#>      
#> 
#> Installing package into '/home/mauro/R/x86_64-pc-linux-gnu-library/4.0'
#> (as 'lib' is unspecified)
#> Adding 'r2dii.data_0.1.4.9000_R_x86_64-pc-linux-gnu.tar.gz' to the cache
packageVersion("r2dii.data")
#> [1] '0.1.4.9000'

devel <- r2dii.data::nace_classification %>%
  filter(code == "A")

typeof(devel$code)
#> [1] "character"
```

<sup>Created on 2020-11-30 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jdhoffa** [comment]
I think this is closed by #228 ya?

**Comment by maurolepore** [comment]
That's right. Thanks!

```r
typeof(r2dii.data::nace_classification$code)
#> [1] "character"
```
